src/types.rs

```diff
@@ -77,6 +77,19 @@ pub enum AccelMode {
     HighResolution,
 }
 
+/// Accelerometer scaling factor
+#[derive(Debug, Clone, Copy, PartialEq)]
+pub enum AccelScale {
+    /// Plus or minus 2g
+    Scale2g,
```

Comment:
The type already has `Scale` in the name so it is a bit redundant to write `AccelScale::Scale2g`. How about something like `AccelScale::G2` ? The starting `G` must be capital, though. Another alternative would be `AccelScale::_2g` but it is a bit cumbersome to write.